app/controllers/CommentsController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+//require_once 'models/CommentModel.php';
+//require_once 'models/PostModel.php';
+namespace app\controllers;
+
+use app\models\CommentModel;
+use app\models\PostModel;
+use app\utils\View;
+
+class CommentsController {
+    private CommentModel $commentModel;
+    private PostModel $postModel;
+
+    public function __construct() {
+        $this->commentModel = new CommentModel(); // Initialize the PostModel instance
+        $this->postModel = new PostModel();
+    }
+
+    public function store($postId) {
```

Comment:
Are the `exit` statements in this function necessary?

Reply:
I think I was using the header function to send header for redirection purposes. So from my understanding to avoid error, ideally every header function should be followed by an exit to terminate. But since you raised this issue, I figured there must be some tricks I don't know about the exit keyword, lol, please let me know

Reply:
Just to clarify a little more - using:

```
header("Location: somepath");
exit;
```

is fine, but i see this are more of a logic issue - is there a way we can minimise the amount of times it is used?